news/2017-06-27-beatmap-spotlight-may-2017.md

```diff
@@ -0,0 +1,124 @@
+
+[![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/kokoronashi.jpg)](https://osu.ppy.sh/s/565432)
+
+Kokoronashi is yet another great mapset contributed by pishifat. Playing the map feels unique, since it avoids rotational flow and focuses entirely on back-and-forth or linear patterns. You should definitely check this one out if you haven't yet!
+
+[![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/yuki-no-hana.jpg)](https://osu.ppy.sh/s/565432)
+
+A calm, beautiful winter song with a very good-looking skin and a very well designed mapset. While the top difficulty is quite spammy in terms of object density, the visual aspect of the map does not fall apart! Combined with great lower difficulties, this mapset deserves to be featured in the spotlights.
+
+## osu!taiko
+
+[![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/classic-pursuit.jpg)](https://osu.ppy.sh/s/373254)
+
+Spooky! Just 7 months after Halloween Hanjamon's Classic Pursuit entered the ranked section, and what a great map it is! Featuring a huge variety of Slider Velocity changes, tricky off-beat patterns, consistency (which does not equal boring) and even 1/8, this map offers a lot of things to enjoy. And if the map is too easy for you: Try it with ANY mod. It's a challenge with Hidden, Hardrock and Doubletime, with the variety of Slider Velocities and complex 1/4+1/8 keeping you on your toes no matter what mod you are using. The set also has nicely done lower diffs, including Hanjamon's signature "Basic" difficulty, which introduces you into the most basic structures of osu!taiko's dons and kats (the red and blue notes!). We hope that you will enjoy this map - and replay it during Halloween!
+
+[![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/proof-of-existance.jpg)](https://osu.ppy.sh/s/427864)
+
+What a high quality set brought up by tasuke912 and SKSalt this month ! What made this chart get featured in the Spotlights is the management of a flawless structure, patterning and flow in each of their respective difficulties, really making a contrast with some other ranked maps this month. tasuke912's pair patterns usage and rhythm changes in the Inner Oni is the signature of a very highly experienced beatmapper, representing the song extremely accurately while making it fun to play through flow and hitsounding.
+
+[![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/on-the-fm.jpg)](https://osu.ppy.sh/s/568544)
+
+This month we have something different in the chart, a gimmick map! Tired of maps with regular sv? Do you want to play maps with constant sv changes? On the FM it's your map! But hey, we are talking of the Inner Oni, right? No! You will have a sv challenge in all diffs, even in the Kantan. Nofool did a good choice with this song as the sv changes fits pretty well and gives a lot of variety, since 1/4 patterns to nonstop 1/3 patterns. What? Too slow? Use HR and you will enjoy more the map!
```

Comment:
I would like to change my comment as Ephemeral suggested to:

+This month we have something different in the chart, a gimmick map! Tired of maps with regular SV (Slider Velocity)? Do you want to play maps with constant SV changes? On the FM it's your map! But hey, we are talking of the Inner Oni, right? No! You will have a SV challenge in all diffs, even in the Kantan. Nofool did a good choice with this song as the SV changes fits pretty well and gives a lot of variety, since 1/4 patterns to nonstop 1/3 patterns. What? Too slow? Use HR and you will enjoy more the map!